zetasql/fuzzing/component/fuzz_targets/prepared_expression_target.cc

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "zetasql/public/evaluator.h"
+#include "zetasql/fuzzing/component/fuzz_targets/prepared_expression_target.h"
+#include "zetasql/fuzzing/component/arguments/argument.h"
+
+namespace zetasql_fuzzer {
+
+void PreparedExpressionTarget::Visit(SQLStringArg& arg) {
+  sql_expression = arg.ReleaseArg();
+}
+
+void PreparedExpressionTarget::Execute() {
+  if (sql_expression) {
+    zetasql::PreparedExpression expression(*sql_expression);
+    expression.Execute();
+  } else {
+    std::cerr << "SQL expression not found" << std::endl;
```

Comment:
Same comment about the logging.

Reply:
Same reasoning above. Again, the fuzz target should be set up at compile time even though the argument extractors are initialized during runtime. Since Argument is polymorphic though we don't have the ability to perform compile time checking.

Reply:
So the thing here is more that the standard logging macros are well behaved & well known inside Google, whereas using a stream here leads to less predictable behavior (around implicit command line flags for controlling logging outputs, etc., etc.).

It should be as simple as including the correct header from ZetaSQL and replacing std::cerr with LOG(ERROR), and would keep us in sync with the rest of the codebase.